update.sh

```diff
@@ -38,6 +38,16 @@ generated_warning() {
 EOH
 }
 
+get_sed_inplace_argument() {
+    echo "foo" > /tmp/sed_check
+    if sed -i"" "s/foo/bar/g" /tmp/sed_check 2>/dev/null; then
+        echo "-i .bak"
+    else
+        echo "-i.bak"
+    fi
+    rm /tmp/sed_check
+}
+
```

Comment:
I did something similar in my own testing, only to find that the macOS (at least 10.13.4) sed works just fine with the `-i.bak` version. So we could just use that everywhere I think. Unless you're aware of some that _don't_ work with that argument format?

Reply:
It works! Thanks!